macros/plugins/snowflake/create_external_table.sql

```diff
@@ -26,14 +26,25 @@
         {%- endfor -%}{%- endif -%}
         {%- if not infer_schema -%}
             {%- for column in columns %}
-                {%- set column_alias = column.alias if column.alias else column.name %}
-                {%- set column_alias_quoted = adapter.quote(column_alias) if column.quote else column_alias %}
                 {%- set column_quoted = adapter.quote(column.name) if column.quote else column.name %}
+                {%- set column_alias -%}
+                    {%- if 'alias' in column and column.quote -%}
```

Comment:
I think this change introduced the bug - https://github.com/dbt-labs/dbt-external-tables/issues/315

Reply:
I'm not sure if this is a bug. left a reply in that issue

Reply:
Ah actually I think I see the issue now. We should be using `column_quoted` instead of `column_alias` here I think?:
https://github.com/dbt-labs/dbt-external-tables/blob/4d26c38c81ac7c39e9ed8f50792b43de769e88cb/macros/plugins/snowflake/create_external_table.sql#L43